### ISSUE TITLE
Integration with YOLO v8

### ISSUE THREAD

**Issue by mo-safwan** [JN-Yang/PConv-SDloss-Data#19]
Dear author, 
Can you provide a detailed overview of how to integrate SD loss in ultralytics/yolo

**Comment by JN-Yang** [comment]
Thanks for your attention。

Steps to integrate SDB loss in ultralytics/yolo:

STEP 1:    Replace our bbox_iou in 'model/loss.py' with 'ultralytics/utils/metrics.py'.

STEP 2:    Change the code in 'ultralytics/utils/loss.py'. (--- means delete, +++ means add)
'''
class BboxLoss(nn.Module):
    ...
    def forward(...):
        """Compute IoU and DFL losses for bounding boxes."""
        weight = target_scores.sum(-1)[fg_mask].unsqueeze(-1)
---        iou = bbox_iou(pred_bboxes[fg_mask], target_bboxes[fg_mask], xywh=False, CIoU=True)
+++        iou = bbox_iou(pred_bboxes[fg_mask], target_bboxes[fg_mask], xywh=False, SDIoU=True, delta=0.5)
'''
